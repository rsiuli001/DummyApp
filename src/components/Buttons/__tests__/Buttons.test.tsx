import React from 'react';
import { shallow } from 'enzyme';
import { Text, TouchableOpacity, View } from 'react-native';
import { Buttons } from '../../';

describe('Buttons', () => {
  describe('rendering', () => {
    const wrapper = shallow(<Buttons />);
    it('renders the container view', () => {
      expect(wrapper).toMatchSnapshot();
      expect(wrapper.find(View).length).toEqual(4);
    });

    it('renders text', () => {
      expect(wrapper.find(Text).length).toEqual(5);
    });

    it('renders touchable opacity', () => {
      expect(wrapper.find(TouchableOpacity).length).toEqual(3);
    });
  });
});
