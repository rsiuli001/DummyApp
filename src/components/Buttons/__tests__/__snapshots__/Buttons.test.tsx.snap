// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons rendering renders the container view 1`] = `ShallowWrapper {}`;
